include/inviwo/core/properties/propertyowner.h

```diff
@@ -56,8 +56,11 @@ class IVW_CORE_API PropertyOwner : public PropertyOwnerObservable,
     PropertyOwner& operator=(const PropertyOwner& that);
     virtual ~PropertyOwner() = default;
 
-    virtual void addProperty(Property* property, bool owner = true);
-    virtual void addProperty(Property& property);
+    virtual Property& addProperty(Property* property, bool owner = true);
+    virtual Property& addProperty(Property& property);
+
+    template <typename... Ts>
+    void addProperties(Ts&... properties);
```

Comment:
:+1: 
Would be nice with a similar method for adding ports

Reply:
The port also requires a name. I'm not sure if it's easily doable to make the argument list inerleaved or whatever...
Also I think you usually don't have so many ports that it's horribly verbose code if you don't have such function (compared to properties).
Any suggestions on how a nice interface should then look like for ports? 
`addPorts(imInport1, "DIffuse Tex", imInport2, "Whatever");` ??

Reply:
The name is for port groups which is only used for ImagePorts (as far as I now). Maybe one can only add to one port group at a time (or the default) like:
```c++
addPorts("GroupName",port1,port2,port3,...);
// or just
addPorts(port1,port2,port3,...); // Default group name used
```
Anyway, I think that would go into another PR

Reply:
That seems reasonable. I will collect some more feedback on my other proposals and then split the whole thing into PRs accordingly ;)